styles/components/avatar.css

```diff
@@ -0,0 +1,9 @@
+.avatar{
+    border-radius: 50%;
+    width: 50px;
+}
+
+.avatar-small{
+    border-radius : 50%;
+    width: 30px;
+}
```

Comment:
Have a common class avatar for both the avatar and the set the border-radius there . So no need for writing border-radius two times !!